app/src/main/java/org/dicekeys/app/encryption/AppKeystore.kt

```diff
@@ -22,12 +24,15 @@ import javax.crypto.spec.IvParameterSpec
  */
 
 class AppKeystore {
-    private var keyStore: KeyStore
 
-    init {
-        keyStore = KeyStore.getInstance(ANDROID_KEYSTORE).apply {
-            load(null)
-        }
+    enum class KeystoreType {
+        BIOMETRIC,
```

Comment:
What do each of these mean?  Do both Face & Finger work for biometric?  How do the others work?

Reply:
BIOMETRIC have the most strict/secure options configured requiring Face or Fingerprint to unlock the cipher object.
AUTHENTICATION required just a device unlock, PIN, pattern etc is acceptable.